web/themes/custom/bevan/sass/components/_facets-checkbox.scss

```diff
@@ -49,7 +49,10 @@
     margin: 1rem 0 1.5rem;
 
     &-row {
+      display: flex;
       line-height: 1.8rem;
+      margin-bottom: 0.45rem;
+
       &__spaced td {
```

Comment:
I guess we can remove this code now?

Reply:
I meant to highlight the td line below

Reply:
Sorry, what bit can be removed? This bit https://github.com/cabinetoffice/national-leadership-centre-drupal/pull/121/files#diff-1ae830a3c7c2ba3843940d9d54be7fd9L53-L55

```scss

      &__spaced td {
        margin-bottom: 0.75rem;
      }
```

?

Reply:
Fixed with a9f5e4e